### DISCUSSION TITLE
我不能用Android7.1.1版本，内核4.4.22+

### DISCUSSION THREAD

**Discussion by luxun4** [bmax121/APatch#577] [Q&A]
![Screenshot_20240801_152301](https://github.com/user-attachments/assets/f307407c-a35a-45e5-acc0-e19a1a905766)
![Screenshot_20240801_152308](https://github.com/user-attachments/assets/9a07b872-fa3c-4ef1-90ee-582ddccd6334)

**Comment by marat2509** [comment]
Are you sure your device has CONFIG_CALLSYMS enabled (and extremely preferably CONFIG_CALLSYMC_ALL), and also the architecture of your device is arm64?
